src/platforms/common/test_stubs/stub_daemon.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+#include <platforms/posix/apps.h>
+
+#include <functional>
```

Comment:
The problem is that we can't use that with ntest on NuttX/STM32.

Reply:
I guess these stubs should go in the POSIX platform then